local_run

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: Alliander N. V.
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+SCRIPT_PATH=$(
+    cd "$(dirname "${BASH_SOURCE[0]}")" || exit
+    pwd -P
+)
+
+images=()
+for dockerfile in $SCRIPT_PATH/dockerfiles/*.Dockerfile; do
+    images+=("$(basename "${dockerfile%.*}")")
+done
+[[ -z $images ]] && echo "No Dockerfiles found" && return 1
+echo "Select image #: "
+select image in "${images[@]}" ; do
+    [[ $image ]] && break
+done
+
+if [ -z "$(docker images -q rcdt\/$image 2> /dev/null)" ]; then
```

Comment:
We kwam er net achter dat we de ros2.Dockerfile moeten veranderen naar rcdt_ros2.Dockerfile, en dan hier een find and replace doen